app/views/addons/index.html.erb

```diff
@@ -0,0 +1,68 @@
+<div class="content">
+  <section class="page-content container-fluid">
+
+    <%= link_to new_addon_path,class:"btn btn-info btn-floating col-md-2 btn-prime" do %>
+      <span>+ Add Addon</span>
+    <% end %></br>
+
+    <div class="row">
+      <div class="col-12">
+        <div class="card">
+          <h5 class="card-header">Data Tables</h5>
+
+          <div class="card-body">
+            <table id="bs4-table" class="table table-striped table-bordered tbl" style="width:100%">
+              <thead>
+                <tr>
+                  <th>Addon #</th>
+                  <th>Title</th>
+                  <th>Price</th>
+                  <th>Available</th>
+                  <th>Actions</th>
+                </tr>
+              </thead>
+
+              <tbody>
+                <% i = 1 %>
+
+                <% @addons.each do |addon|%>
+                  <tr>
+                    <td><%= i %></td>
+                    <td><%= addon.title.titleize %></td>
+                    <td><%= addon.price %></td>
+                    <td>
+                      <span class="<%= addon.is_available ? 'badge badge-primary': "badge badge-secondary" %>">

```

Comment:
Create a view helper for it for example
in app/helpers/addon_helper.rb
`def badge_class(addon)`
`   return '' unless addon.present?`
`
   addon.is_available ? 'badge badge-primary': "badge badge-secondary"`
` end`
`

Reply:
I'm creating this helper in app/helpers/application_helper  instead of app/helpers/addon_helper because this helper will be used for deals, food items, group items, add-ons and menus along with add-ons.